packages/design/src/colors.css

```diff
@@ -58,35 +58,39 @@
   --color-overlay--dimmed: rgba(var(--color-white--rgb), 0.6);
 
   /* Brand */
-  --color-brand: var(--color-green);
+  --color-brand: rgb(124, 175, 14);
   --color-brand--highlight: rgb(132, 234, 0);
 
   /* Jobber Workflow Colors */
-  --color-request: var(--color-indigo);
-  --color-quote: var(--color-pink);
-  --color-job: var(--color-yellowGreen);
-  --color-visit: var(--color-green);
+  --color-request: rgb(185, 133, 70);
```

Comment:
These new "workflow" colours are largely one-offs that do not need to reference an underlying "base" palette.